_data/meltano/utilities/dbt-osmosis/z3z1ma.yml

```diff
@@ -64,6 +41,9 @@ next_steps: |
 
   4. Ensure you additionally add `adapter: ...` to your dbt-osmosis config for the built
   in commands to work. ie for Bigquery, `adapter: bigquery`
+
```

Comment:
Sure thing!

You have to modify the pip url so that your specific adapter package is included. To my knowledge, mutating the pip url causes the hub commands not to be found. I had a user report this on the dbt osmosis repo and I recall the same experience. 

So the best recommendation I can think of is to add the commands directly from GitHub to their own yaml if they choose to use them. 

If this is no longer the case, we can drop it.